qemu/tests/cfg/mq_enabled_chk.cfg

```diff
@@ -8,6 +7,13 @@
     type = mq_enabled_chk
     vhost = on
     image_snapshot = yes
-    smp = 4
-    queues_list = 8 4 2
-    start_vm = no
+    timeout = 360
+    driver_name = "netkvm"
+    Linux:
+        start_vm = no
+        smp = 4
+        queues_list = 8 4 2
+    Windows:
+        type = mq_enabled_chk_windows
+        queues = ${smp}
```

Comment:
Does this smp refer to the smp defined in Linux part?

Reply:
For linux part, this case need different queues to assign, but for windows we need not change queues (smp=queues), so need not to fix the smp value.